packages/utilities/psammead-rich-text-transforms/src/candy-xml.js

```diff
@@ -44,13 +44,25 @@ const createUrlLink = element => {
   return urlLink(text, locator, blocks);
 };
 
+const handleSupportedNodes = (childNode, attributes, acc) => {
+  const block = xmlNodeToBlock(childNode, attributes);
+  const blocks = Array.isArray(block) ? block : [block];
```

Comment:
Howcome the switch from `is(Array, block)`?

Reply:
Weirdly it failed locally for me in the tests but now it's fine, will swap out